extensions/wikia/WikiaApi/WikiaApiQuerySiteinfo.php

```diff
@@ -66,7 +66,15 @@ protected function appendVariables($property) {
 		$data = array ();
 		foreach ($this->variablesList as $id => $variableName) {
 			$data[$id] = array( 'id' => $variableName );
-			$value = (array_key_exists($variableName, $GLOBALS) && !is_null($GLOBALS[$variableName])) ? $GLOBALS[$variableName] : "";
+			switch ($variableName) {
+				case 'wgWikiaGlobalUserGroups':
+					/** @var \Wikia\Service\User\Permissions\PermissionsServiceImpl $permissionsService */
```

Comment:
I think we should try and use the interface rather than the implementation:

```php
/** @var \Wikia\Service\User\Permissions\PermissionsService $permissionsService */
```